services/delegationAnalysis/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "taraxa-scripts",
```

Comment:
To be consistent, the name should be:
@taraxa_project/delegation-analysis
Then you need to add this into the root tsconfig.json insite paths and references